src/Providers/ViewServiceProvider.php

```diff
@@ -25,14 +27,33 @@ public function register()
             return new Cascade($app['request'], Site::current());
         });
 
+        $antlersVersion = config('statamic.antlers.version', 'regex');
```

Comment:
I think since the intention is for this parser to eventually become the default, we should test it that way.

In `tests/TestCase.php` at the end of `resolveApplicationConfiguration` you can add this:

```
$app['config']->set('statamic.antlers.version', 'runtime');
```

(The other config bootstrapping we're doing in `getEnvironmentSetup` should really be in there too, but we'll fix that up separately)


When that's done and you run the tests, you'll see failures.

Reply:
I've added this to the PR myself. 👍